sequencer/src/block/full_payload/ns_table.rs

```diff
@@ -237,6 +317,16 @@ impl Committable for NsTable {
     }
 }
 
+/// Return type for [`NsTable::validate`].
+///
+/// TODO wtf is our error handling pattern?
```

Comment:
The new convention, that we are aiming to stick with, is to use `thiserror` when we want to distinguish different types of errors in code (even in tests) and use `anyhow` otherwise. So this should be good

Reply:
Thanks. Comment removed in https://github.com/EspressoSystems/espresso-sequencer/pull/1607/commits/71583bde78b58123adbadd55fca83c1847a094c4.